incubator/hnc/e2e/tests/e2e_test.go

```diff
@@ -39,6 +39,30 @@ func tryRun(cmdln ...string) error {
 	return err
 }
 
+func runShouldContain(substrs []string, timeDuration string, cmdln ...string) {
```

Comment:
Might be less ambiguous for `timeDuration string` to be `seconds float64` (see http://onsi.github.io/gomega/#eventually).

Reply:
Also, why do we need multiple substrings? That adds a lot of visual complexity to the caller (`[]string{X}` where you really only needed `X`) in the vast majority of cases. Maybe just accept a single string, and if we _also_ need a "multi" version, add a `runShouldContainMultiple` function?

Reply:
done

Reply:
we need the multiple version for 
<pre><code>
// Tree should show CannotPropagateObject in nsParent and CannotUpdateObject in nsChild
runShouldContainMultiple([]string{"1) CannotPropagateObject", "2) CannotUpdateObject"}, 1, "kubectl hns tree", nsParent)
</code></pre>